src/ui/components/router-links/RouterLinks.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from 'react';
+import { Link } from 'react-router-dom';
+import * as shortid from 'shortid';
+
+import Button from '@material-ui/core/Button';
+
+import { NavigationLink } from '../../../interfaces';
+
+interface RouterLinksProps {
+  links: NavigationLink[];
+}
+
+const RouterLinks: FunctionComponent<RouterLinksProps> = ({ links }) => {
+  return (
+    <React.Fragment>
+      {links.map((link) => (
+        <Button
+          color="inherit"
+          key={'router-link' + shortid.generate()}
```

Comment:
awesome

Reply:
not sure how big the lib is, is it possible to destruct generate instead of *?

Reply:
No biggie. You don't have to make the changes, but noticed that we can also destruct Fragment from react. name and target from links array as well. `links.map((({name, target}))`

Reply:
done 👍